scholia/app/templates/venue.html

```diff
@@ -352,5 +361,9 @@ <h3>Authorships</h3>
 <table class="table table-hover" id="authorships-gender-distribution"></table>
 
 
+<h2 id="AuthorAwards">Author Awards</h2>
```

Comment:
I am wondering whether there should be a space in "AuthorAwards"?

Reply:
That's just the id of the header, right? The text itself has a space.

Reply:
Yes. On the other hand, I do not know if it is a good idea to have spaces in the ids (as I have been doing in the past). Do you know if one can run into problems with that? Maybe it is better with a dash?

Reply:
Ah okay. I never used spaces in ids. 
Just found this for example: "HTML 5 is even more permissive, saying only that an id must contain at least one character and may not contain any space characters."

So I guess it's better to remove spaces. It doesn't really matter if it's author-awards or AuthorAwards or authorAwards but I would try to stick to one naming convention.

Reply:
Thanks for the lookup. Maybe we should convert the spaces to a dash and lowercase when the word is not the first, e.g., "Author-awards". I hope that does not collide with the table-id.